packages/create-svelte/templates/default/.gitignore

```diff
@@ -1,12 +1,17 @@
-.DS_Store
-node_modules
-/build
-/.svelte-kit
-/package
-.env
```

Comment:
If I recall, the reason this was ignored was that people were worried about accidentally checking in production secrets. It makes sense to me to ignore it because you're not going to check it in here, but will instead provide it as part of your production deploy

Reply:
Ignoring the `.env` file goes agaist the Vite convention.

https://vitejs.dev/guide/env-and-mode#env-files

```
.env                # loaded in all cases
.env.local          # loaded in all cases, ignored by git
.env.[mode]         # only loaded in specified mode
.env.[mode].local   # only loaded in specified mode, ignored by git
```

Since SvelteKit is at it core a Vite project, following this seemed intuitive.

Especially since `.env.test` and `.env.test.local` files are utilized by Vitest.